pkg/psmdb/mongo/mongo.go

```diff
@@ -477,9 +507,22 @@ func (m *ConfigMembers) SetVotes() {
 		if member.Hidden {
 			continue
 		}
+
+		if _, ok := member.Tags["external"]; ok {
+			[]ConfigMember(*m)[i].Votes = member.Votes
+			[]ConfigMember(*m)[i].Priority = member.Priority
+
+			if member.Votes > 0 {
+				votes++
```

Comment:
can `member.Votes` be more than one?
As i see from the code below, we use `votes++` because we set `member.Votes = 1`

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/729/commits/db744b631bde51f6969c4340d944ef36d2ce2b10